omero/sysadmins/python3-migration.rst

```diff
@@ -0,0 +1,128 @@
+   `Choosing a platform`_ below.
+2. Install OMERO.server and OMERO.web *separately*. Though not necessary, all
+   instructions like `OMERO.server`_ and `OMERO.web`_ below as well as the main
+   :doc:`server <unix/server-installation>` and 
+   :doc:`web <unix/install-web/web-deployment>` installation pages now assume
+   that the two are in separate installations.
+3. Once both have been installed, perform a
+   :doc:`backup and restore <server-backup-and-restore>` procedure
+   and test your installation against the copy of your data.
+
+Choosing a platform
+-------------------
+
+We suggest Python 3.6. This is the default installation on the two recommended platforms,
+CentOS 7 and Ubuntu 18.04.
+
+Both 3.5 and 3.7 should work and are slated to have support added, but Python 3.6 has been
+the focus of testing during the migration.
+
+Similarly, other operating systems are slated for having support added, but help from the
+community would be very welcome! Obvious next candidates are CentOS 8 and Ubuntu 20.04.
+
+Debian is more of an issue. Debian 9 is still on Python 3.5 and Debian 10 has moved to Ice 3.7
+so neither is particularly well supported at the moment. We are examining workarounds like
+building Ice from source and/or building a .deb package.
+
+Other prerequisites
+-------------------
+
+OMERO's other prerequisites have not changed substantially but if you would like to take this
+opportunity to move to the :doc:`recommended version<version-requirements>` for all requirements,
+the current choices are:
+
+- Ice 3.6 (non-optional)
+- Java 11
+- Nginx 1.14
+- PostgreSQL 11
+
+Other options
+-------------
+
+The installation walkthroughs provided in the documentation try to stick to a minimum installation.
+The only requirements are an understanding of the Unix shell, the standard package manager for your
+platform, and the regular Python distribution mechanisms.
+
+However, more advanced installation mechanisms are available if you are interested and have familiarity
+with the given mechanism:
+
+- `Ansible roles <https://galaxy.ansible.com/ome>`_ are available for most installation steps. The primary
+  roles, `omero-server` and `omero-web` have not yet been released and will need to be installed from
+  GitHub.
+
+- A `conda channel <https://anaconda.org/ome>_` provides pre-built packages needed by OMERO if you are
+  comfortable not using your platforms standard Python installation.
+
+- `Docker images <https://hub.docker.com/u/openmicroscopy>`_ are also available. Both the `omero-server`
+  and `omero-web` images are considered production quality.
+
+Please get in touch at https://forum.image.sc/c/data if you have any questions.
+
+OMERO.server
+------------
+
+The steps for an OMERO.server installation have not changed substantially.
+
+Download the OMERO.server.zip as you would usually do, and unpack it under your
+installation directory. We suggest: file:`/opt/omero/server/OMERO.server`
```

Comment:
```suggestion
installation directory. We suggest: file:`/opt/omero/server/` and symlink the unpacked
directory to `OMERO.server`
```

Reply:
That is not the case for omero-install we still have it under /home/omero
That will need to be adjusted

Reply:
sorted in omeroweb-install and omero-install